srcpkgs/gnuradio/template

```diff
@@ -1,28 +1,40 @@
 # Template file for 'gnuradio'
 pkgname=gnuradio
-version=3.8.0.0
-revision=9
+version=3.10.0.0
+revision=1
 build_style=cmake
 conf_files="/etc/gnuradio/conf.d/*"
-configure_args="-DENABLE_INTERNAL_VOLK=OFF -DGR_PYTHON_DIR=/${py3_sitelib}"
-hostmakedepends="pkg-config doxygen swig python3-Mako python3-Sphinx
- python3-cairo python3-numpy python3-yaml texlive texlive-latexextra texlive-dvi"
-makedepends="SDL-devel boost-devel fftw-devel gsl-devel jack-devel
- libgsm-devel libusb-devel volk-devel uhd-devel gtk+3-devel python3-devel
- python3-gobject-devel log4cpp-devel gmpxx-devel mpir-devel"
-depends="python3-cheetah3 python3-numpy python3-Mako python3-gobject
- python3-yaml"
-checkdepends="python3-scipy"
+configure_args="-DZEROMQ_INCLUDE_DIRS=${XBPS_CROSS_BASE}/usr/include
+ -Dpybind11_DIR=/${py3_sitelib}/pybind11/share/cmake/pybind11"
+# TODO: add python3-pygccxml (not packaged in Void)
+_pydeps="python3-Mako python3-cairo python3-click python3-click-plugins python3-gobject
+ python3-numpy python3-pyqtgraph python3-pyzmq python3-scipy python3-yaml"
+hostmakedepends="pkg-config doxygen python3-Sphinx python3-pybind11 ${_pydeps}
+ texlive texlive-latexextra texlive-dvi"
+makedepends="SDL-devel SoapySDR-devel boost-devel codec2-devel cppzmq fftw-devel gmpxx-devel
```

Comment:
Any chance it can use SDL2 instead?

Reply:
No, they use SDL_Overlay is no longer available in SDL2: 
```console
$ git clone https://github.com/gnuradio/gnuradio
$ cd gnuradio && grep -Rhow 'SDL_[A-Z][a-z][a-zA-Z]*' gr-video-sdl/lib/ | sort | uniq
SDL_CreateYUVOverlay
SDL_Delay
SDL_DisplayYUVOverlay
SDL_Flip
SDL_FreeYUVOverlay
SDL_GetError
SDL_GetTicks
SDL_Init
SDL_LockYUVOverlay
SDL_Overlay
SDL_Quit
SDL_Rect
SDL_SetVideoMode
SDL_Surface
SDL_UnlockYUVOverlay
```